service/matching/tasklist/matcher.go

```diff
@@ -60,20 +61,15 @@ type TaskMatcher struct {
 	numPartitions func() int    // number of task list partitions
 }
 
-const (
-	_defaultTaskDispatchRPS    = 100000.0
-	_defaultTaskDispatchRPSTTL = 60 * time.Second
-)
-
 // ErrTasklistThrottled implies a tasklist was throttled
 var ErrTasklistThrottled = errors.New("tasklist limit exceeded")
 
 // newTaskMatcher returns an task matcher instance. The returned instance can be
 // used by task producers and consumers to find a match. Both sync matches and non-sync
 // matches should use this implementation
-func newTaskMatcher(config *taskListConfig, fwdr *Forwarder, scope metrics.Scope, isolationGroups []string, log log.Logger) *TaskMatcher {
-	dPtr := _defaultTaskDispatchRPS
-	limiter := quotas.NewRateLimiter(&dPtr, _defaultTaskDispatchRPSTTL, config.MinTaskThrottlingBurstSize())
+func newTaskMatcher(config *config.TaskListConfig, fwdr *Forwarder, scope metrics.Scope, isolationGroups []string, log log.Logger) *TaskMatcher {
+	dPtr := config.TaskDispatchRPS
```

Comment:
Is this what you were referring to as having to be pulled into a central location?

Reply:
the central location of the default values is config/config.go